proto/servicemanager/v3/servicemanager.proto

```diff
@@ -76,11 +76,12 @@ message VersionInfo {
 }
 
 message InstanceInfo {
-    InstanceIdent instance = 1;
-    uint32 uid             = 2;
-    uint64 priority        = 3;
-    string storage_path    = 4;
-    string state_path      = 5;
+    InstanceIdent instance      = 1;
+    uint32 uid                  = 2;
+    uint64 priority             = 3;
+    string storage_path         = 4;
+    string state_path           = 5;
+    bytes network_configuration = 6;
 }
```

Comment:
maybe just remove two commits "Add network conf field to InstanceInfo" and  "revert Add network conf field to InstanceInfo" ?

Reply:
It is merge of feature branch to develop. These commits are in feature branch already. Shall we change some commits during merge?